src/CoreLoadEstimator.h

```diff
@@ -67,8 +67,9 @@ class CoreLoadEstimator {
     /*
      * We will attempt to increase the number of cores if the load factor
      * increases beyond this threshold.
+     * 1.5 is the one we used in the paper.
```

Comment:
I'm not sure it makes sense to justify these numbers in the documentation by saying we used them in the paper; it doesn't really make them any less arbitrary because they're in the paper. I'd probably remove both this comment and the one on line 91.

Reply:
Sure I can remove that.